Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM perl:5.30.2-slim-threaded
+
+RUN apt-get update && \
+    apt-get install -yq \
+      libcurses-perl \
+      libjson-perl \
+      libstatistics-descriptive-perl && \
+    apt-get clean && \
+    rm -rf \
+	    /tmp/* \
+	    /var/lib/apt/lists/* \
+	    /var/tmp/*
```

Comment:
I'm a little scared by this. Why is it required? Can we skip the "rm -rf" part?

Reply:
Those lines are only present to properly cleanup any temporary files generated during the `apt-get update` and `apt-get install` commands.

Below is a comparison of the base container, the container without cleanup, and the container with cleanup.

```
$ docker images
REPOSITORY              TAG                    IMAGE ID            CREATED             SIZE
chrislwade/ping-multi   no-cleanup             ee946270d1a0        2 seconds ago       190MB
chrislwade/ping-multi   latest                 d1e2ca58b026        25 hours ago        173MB
perl                    5.30.2-slim-threaded   38713c92fcb5        2 weeks ago         125MB
```

And to be clear, this is the actual files being removed.

```
-rw-r----- 1 root root        0 May 10 03:30 /var/lib/apt/lists/lock
-rw-r--r-- 1 root root    49287 May 10 02:10 /var/lib/apt/lists/deb.debian.org_debian_dists_buster-updates_InRelease
-rw-r--r-- 1 root root    65372 May  9 14:14 /var/lib/apt/lists/security.debian.org_debian-security_dists_buster_updates_InRelease
-rw-r--r-- 1 root root   121474 May  9 10:04 /var/lib/apt/lists/deb.debian.org_debian_dists_buster_InRelease
-rw-r--r-- 1 root root 16782044 May  9 08:56 /var/lib/apt/lists/deb.debian.org_debian_dists_buster_main_binary-amd64_Packages.lz4
-rw-r--r-- 1 root root   379416 May  8 19:09 /var/lib/apt/lists/security.debian.org_debian-security_dists_buster_updates_main_binary-amd64_Packages.lz4
-rw-r--r-- 1 root root    12411 Feb 23 19:59 /var/lib/apt/lists/deb.debian.org_debian_dists_buster-updates_main_binary-amd64_Packages.lz4
```

With Docker images it is normal to ensure nothing beyond what is necessary for the function of the container is present in the final container image.  This is especially true when the source image is a "slim" image.  By comparison, this image is just under 700MB when using the normal perl image.